app/cafe.py

```diff
@@ -0,0 +1,21 @@
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+import datetime
```

Comment:
correct way to import modules:
1. Standard library imports
2. Third-party imports
3. Application-specific imports